django_celery_fulldbresult/admin.py

```diff
@@ -1,3 +1,13 @@
 from django.contrib import admin
 
+from django_celery_fulldbresult.models import TaskResultMeta
+
+
+class TaskResultMetaAdmin(admin.ModelAdmin):
+    list_display = ("task_id", "task", "date_submitted", "date_done", "status")
```

Comment:
Seems to me there is 1 more field here than in the given screenshot, maybe update the screenshot?